### ISSUE TITLE
SNMP page 

### ISSUE THREAD

**Issue by gtmills** [openbmc/webui-vue#33]
We had an SNMP panel on phosphor-webui. 
Need the SNMP Redfish API first. 
SNMP was added to Redfish in 2019.4 just need to implement it in bmcweb.

**Comment by derick-montague** [comment]
Blocked by BMCWeb implementation.

**Comment by macpijan** [comment]
Hello. Can I ask what is the status here and where is the blocker (issue link)? What would be the next step to move it forward? Thanks.

**Comment by derick-montague** [comment]
> Hello. Can I ask what is the status here and where is the blocker (issue link)? What would be the next step to move it forward? Thanks.

Yes, there is discussion and consensus for a path forward in Gerrit.
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/38599

**Comment by derick-montague** [comment]
SNMP has been moved to a page called alerts, which is under review in Gerrit waiting for backend changes.

**Comment by gtmills** [comment]
https://gerrit.openbmc.org/c/openbmc/bmcweb/+/38599 are the backend changes
